app/v2/inbound_sms/inbound_sms_schemas.py

```diff
@@ -0,0 +1,77 @@
+from app.schema_validation.definitions import uuid
+
+
+get_inbound_sms_request = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "schema for query parameters allowed when getting list of received text messages",
+    "type": "object",
+    "properties": {
+        "older_than": uuid,
+    },
+    "additionalProperties": False,
+}
+
+
+get_inbound_sms_single_response = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "GET inbound sms schema response",
+    "type": "object",
+    "title": "GET response v2/inbound_sms",
+    "properties": {
+        "provider_date": {
+            "format": "date-time",
+            "type": "string",
+            "description": "Date+time sent by provider"
+        },
+        "provider_reference": {"type": ["string", "null"]},
+        "user_number": {"type": "string"},
+        "created_at": {
```

Comment:
Let's just uses either created_at or provider_date, having both may be confusing for the service.

Reply:
ok, created_at makes more sense

Reply:
will we ever need the provider info to be serialized? as I could just update the `serialize` function to strip those out otherwise will create another serialize function for api calls.